jekyll/_cci2/contexts.md

```diff
@@ -21,13 +21,15 @@ Contexts are created on the Settings page of the CircleCI application, in the Or
 
 To use environment variables set on the Contexts page, the person running the workflow must be a member of the organization for which the context is set and the rule must allow access to all projects in the org. The default context name is `org-global` and the default rule allows access to all projects in the org.
 
+Context names must be unique for each Github or Bitbucket organization. For CircleCI installations on your own server or private cloud, Context names must be unique across the installation.
```

Comment:
@AlexanderMann The second sentence above is incorrect, right?

Reply:
I don't _think_ so...can't Server installations only have a _single_ Organization? It'd have the same effect...

Reply:
@smaant maybe you can review this as well, please